### PR TITLE
Adding debug and error statements

### PR DESCRIPTION
**Pull request by nkoenig**

### CODE REVIEW

File: ros1_ign_bridge/include/ros1_ign_bridge/factory.hpp [hunk 1]
```diff
@@ -114,7 +115,7 @@ class Factory : public FactoryInterface
     const boost::shared_ptr<ros::M_string> & connection_header =
```
Comment (chapulina) [inline ros1_ign_bridge/include/ros1_ign_bridge/factory.hpp]: I was looking at this function on `ros1_bridge` yesterday and noticed they [print more](https://github.com/ros2/ros1_bridge/blob/master/include/ros1_bridge/factory.hpp#L148-L188) debug info than we do. We could do the same here, using the type names:

**Comment by caguero** [comment]
Should we add `rosconsole` as a new dependency? (`package.xml`, `CMakeLists.txt`, `README.md`)

**Comment by chapulina** [comment]
> Should we add rosconsole

We're getting it indirectly through `roscpp`, but it wouldn't hurt to have the direct dependency :+1:

File: ros1_ign_bridge/include/ros1_ign_bridge/factory.hpp (hunk 1 above)
Comment (nkoenig) [reply ros1_ign_bridge/include/ros1_ign_bridge/factory.hpp]: 90ce9939f7c0218290f7c21fd20a0971a845cbaa

**Comment by nkoenig** [comment]
Added rosconsole: 90ce9939f7c0218290f7c21fd20a0971a845cbaa

**Pull request merged**